docs/setupDevEnvironmentUbuntu.md

```
@@ -127,13 +127,12 @@ mkdir runtime
 * Change `EXPRESS_SESSION_SECRET_KEY` to a random string with no space
 * Change `PROJECT_CONFIG` to point to your project's configuration file. The default is an example configuration file that can be copied and configured for your own need.
 
+The default map uses OpenStreetMap tiles which don't require any API keys. If you want to add custom raster tiles (such as aerial imagery), you can optionally configure:
+* `CUSTOM_RASTER_TILES_XYZ_URL=https://your-tile-server/{z}/{x}/{y}` - URL template for your custom tile server
+* `CUSTOM_RASTER_TILES_MIN_ZOOM=10` - Minimum zoom level at which custom tiles are available (default: 0)
+* `CUSTOM_RASTER_TILES_MAX_ZOOM=18` - Maximum zoom level at which custom tiles are available (default: 22)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)